src/test/java/com/fasterxml/jackson/databind/StringDeserializerTest.java

```diff
@@ -0,0 +1,68 @@
+package com.fasterxml.jackson.databind;
+
+import com.fasterxml.jackson.databind.testutil.DatabindTestUtil;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+/**
+ * Test validation uses a string to accept JSON substrings
+ * instead of throwing exceptions by default
+ */
+public class StringDeserializerTest
+{
+
+    @Test
+    public void acceptSubJsonTest() throws Exception {
+        String json = "{\"name\":\"root\"," +
```

Comment:
* use the method a2q which converts apostrophes to double quotes
* the text then should use apostrophes instead of quotes - the advantage is that you don't need to escape apostrophes making the text easier to read
* example - https://github.com/FasterXML/jackson-databind/blob/70deb562a742ce36a7fceaa6f3c330ba9334ccbe/src/test/java/com/fasterxml/jackson/databind/RootNameTest.java#L60